main/Helper.cpp

```diff
@@ -478,7 +478,7 @@ double ConvertTemperature(const double tValue, const unsigned char tSign)
 	return RoundDouble(ConvertToFahrenheit(tValue),1);
 }
 
-std::vector<std::string> ExecuteCommandAndReturn(const std::string &szCommand)
+std::vector<std::string> ExecuteCommandAndReturn(const std::string &szCommand, int *returncode)
```

Comment:
please change this to:
std::vector<std::string> ExecuteCommandAndReturn(const std::string &szCommand, int &returncode)